test-suite/testkit/Cargo.toml

```diff
@@ -44,6 +44,7 @@ pretty_assertions = "0.6.1"
 assert_matches = "1.2.0"
 thread_local = "1.0.0"
 lazy_static = "1.0.0"
+exonum-cryptocurrency = { version = "0.0.0", path = "../../examples/cryptocurrency" }
```

Comment:
Hmmm... `exonum-testkit` is a dev-dependency of the `cryptocurrency` service. In theory nothing should break, but just to be sure, let's create a simplest mock service in `tests.rs` instead of new dependency?